packages/components/pages/f-mfa/stories/story.helper.js

```diff
@@ -0,0 +1,98 @@
+import {
+    SUBMIT_LOGIN_CHALLENGE_URL
+} from '../src/constants';
+
+const httpStatusCodes = {
+    ok: 200,
+    badrequest: 400,
+    throttledrequest: 429
+};
+
+const httpVerbs = {
+    post: 'POST'
+};
+
+// Available states for the API mock
+const apiStates = {
+    none: 'no-issues',
+    apiPostFailed400: 'post-otp-fails-due-to-bad-request',
+    apiPostFailed429: 'post-otp-fails-due-to-throttled-request'
+};
+
+// Mocks for the API calls
+const otpPOST200 = {
```

Comment:
Do you need these definitions? You could just pass them as objects directly into the `apiDefinitions`? Will be all in once place and no need for a const.

Reply:
Because in the future (and in other examples) you may want to mix and match GET/POST scenarios and assign them to different definitions ... so if you created them within the definitions then you would be duplicating code.